modules/aws_vpc/variables.tf

```diff
@@ -107,6 +122,21 @@ variable "region" {
 }
 
 variable "security_group_id" {
-  description = "Security Group ID"
+  description = "Security Group ID local variable value"
+  type        = string
+}
+
+variable "raw_sec_group_id" {
```

Comment:
Why do we have local vs raw? Again, seems we're creating variables and items to maintain when they are not needed.

Reply:
I needed a way to know if the `security_group_id` input variable had a value set or not to determine which `byo_network_scenario` is in effect. The current local.security_group_id hides that information since it is a derived value and could also be set to the IaC newly created security group resource. I left `local.security_group_id` as is, and created a new module input to get the uncooked security_group_id input variable value so I could check if an existing security group id value was provided as an input variable or not. If the name is confusing and renaming the module input variable to something else would help, I am open for suggestions.